src/back-end/src/app.js

```diff
@@ -53,6 +56,10 @@ function sortBySkillsThenTimezone(array1, array2) {
   return 0;
 }
 
+/** Returns true if a given user is a mentee, in the same organization as
+ * the given mentor, and is neither accepted not declined by the given mentor.
```

Comment:
Very minor: I could be reading this wrong, but I think you may have meant to say 'nor' rather than 'not'
```suggestion
 * the given mentor, and is neither accepted nor declined by the given mentor.
```